content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -146,7 +326,7 @@ Datadog consolidates and applies additional tags from various sources to cost me
 
```

Comment:
In the above table for `Out-of-the-box tag` I think we should rework it to just be a table that describes the `orchestrator` tag. (dropping all the other rows as they are becoming redundant with the rest of the page)

The orchestrator section should say something about how we add a `orchestrator` tag to all resources associated with your cluster. 

If there could be a tab per cloud here as well. Azure and GCP will have the table with a single row for `orchestrator:kubernetes` while AWS will also have the `orchestrator:ecs` row